src/main/java/uk/gov/ons/census/fwmt/canonical/v1/CreateFieldWorkerJobRequest.java

```diff
@@ -6,33 +6,52 @@
 import lombok.NoArgsConstructor;
 
 import java.io.Serializable;
-import java.time.LocalDate;
-import java.util.Map;
 import java.util.UUID;
 
 @Data
 @AllArgsConstructor
 @NoArgsConstructor(access = AccessLevel.PUBLIC)
 public class CreateFieldWorkerJobRequest implements Serializable {
 
-  private String actionType;
+  private UUID caseId;
+
+  private String caseReference;
 
-  private String jobIdentity;
+  private String caseType;
 
   private String surveyType;
 
-  private UUID caseId;
+  private String establishmentType;
 
-  private boolean preallocatedJob;
+  private String mandatoryResource;
 
-  private String mandatoryResourceAuthNo;
+  private String coordinatorCode;
 
-  private LocalDate dueDate;
+  private String coordinatorId;
 
-  private Address address;
+  private String uua;
+
+  private boolean sai;
+
+  private String description;
+
+  private String specialInstructions;
+
+  private String ccsQuestionnaireURL;
```

Comment:
You need to check the types against the following, I think some should be int and boolean 
https://share.sp.ons.statistics.gov.uk/sites/dsttd/fmta/FWMT/Shared%20Documents/FWMT_Working_Area/Solution%20Architecture/Census/Mapping/ObjectDefs.ods